File: Demais/CalculoSimples.cpp
// CalculoSimples

#include <iostream>
#include <iomanip>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int codum;
int numpecaum;
float valorpecaum;

int coddois;
int numpecadois;
float valorpecadois;

float soma;

using namespace std;

int main() {

cin>>codum>>numpecaum>>valorpecaum;
cin>>coddois>>numpecadois>>valorpecadois;

soma = (numpecaum*valorpecaum)+(numpecadois*valorpecadois);

cout<<fixed<<setprecision(2)<<"VALOR A PAGAR: R$ "<<soma<<endl;


}

File: Demais/CalculoSimples.py
# CalculoSimples

soma = float

codum,numpecaum,valorpecaum, = input().split(" ")
codum = int (codum)
numpecaum = int (numpecaum)
valorpecaum = float (valorpecaum)

coddois,numpecadois,valorpecadois, = input().split(" ")
coddois = int (coddois)
numpecadois = int (numpecadois)
valorpecadois = float (valorpecadois)

soma = (numpecaum*valorpecaum)+(numpecadois*valorpecadois)

print("VALOR A PAGAR: R$ %.2f"%soma)




